arches/app/search/search_export.py

```diff
@@ -391,7 +394,12 @@ def flatten_tiles(self, tiles, datatype_factory, compact=True, use_fieldname=Fal
                 node = self.get_node(nodeid)
                 if node.exportable:
                     datatype = datatype_factory.get_instance(node.datatype)
-                    node_value = datatype.get_display_value(tile, node)
+                    if self.export_system_values:
+                        node_value = datatype.transform_export_values(
+                            value, **{"concept_export_value_type": "id"}
```

Comment:
Just a note - 'concept_export_value_type' is used here because that's the argument in both the domain and concept datatypes. Might be nice to use something more generic in the controlled list manager.